types/cannon/index.d.ts

```diff
@@ -858,11 +858,11 @@ declare module CANNON {
         constructor(vertices: number[], indices: number[]);
 
         updateTree(): void;
-        getTrianglesInAABB(aabb: AABB, result: []): [];
+        getTrianglesInAABB(aabb: AABB, result: number[]): [];
```

Comment:
Optional:

The only thing I would suggest changing is this line. At the moment the return type `[]` says that the function returns an empty tuple (an empty array), not `unknown[]` or `any[]`.

I would opt for changing it to return `unknown[]` so it's easier to work with :smile: 

```suggestion
        getTrianglesInAABB(aabb: AABB, result: number[]): unknown[];
```

To see why it's a problem, take a look at https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEAzArgOzAFwJYHtVJxwAoBKALngG0BdAbgCh6w8BnDKqAGngCNuxq8ALwFiJWkA